### ISSUE TITLE
Error in Patch file

### ISSUE THREAD

**Issue by Karmugilan9345** [insidesherpa/bank-mergeCoTechTask1#261]
I got the patch file in the sidebar, but when I download the whole file in zip format patch file does not exist in the zip folder.
I got an error while executing this line of code mv 0001-Create-Patch-File.patch
Check the below link for the screenshot of the error

https://user-images.githubusercontent.com/55269099/126949483-4c284783-f069-4756-b9c4-cd8e98f8d309.png

**Comment by devinl8672** [comment]
Did you get this resolved? I also can't find my patch file

**Comment by Karmugilan9345** [comment]
> 
> 
> Did you get this resolved? I also can't find my patch file

Yes, I got the patch file. Try the REPL environment it was easier(for the 1st task).

**Comment by devinl8672** [comment]
Yeah I eventually figure out that I can just download the patch file and submit that. Thanks for your help anyway.
